corehq/apps/export/models/new.py

```diff
@@ -1006,6 +1009,15 @@ def _generate_schema_from_case_property_mapping(case_property_mapping, app_id, a
         return schema
 
     @staticmethod
+    def _generate_schema_for_parent_case(app_id, app_version):
```

Comment:
https://github.com/dimagi/commcare-hq/pull/10984/commits/40656e3d46891808cd30f0efa8181d7418d47056 punting for now, but will add